ReleaseNotes-2.9

```diff
@@ -96,7 +110,10 @@ Wanted changes:
 Other changes
 =============
 
- *
+ * Rubocop agreement, cleanup rubocop configuration and apply missing cops.
+ * Refactor of Event module.
+ * Move ProjectLogRotate to 1 event per job.
+ * Frontend test suite migration of several tests from minitests to rspec.
```

Comment:
This has no consequences for the admin, so it does not belong here.

Reply:
When you say "it" what are you referring to exactly? The last line or all the four lines?

When you say that "This has no consequences for the admin" do you mean that the section "Other changes" is used for... something related to "consequences for the admin"? Where do you suggest this four lines can be placed into?